snippets/dart_snippets/lib/lnurl_withdraw.dart

```diff
@@ -3,21 +3,22 @@ import 'package:breez_sdk/bridge_generated.dart';
 
 Future<void> lnurlWithdraw() async {
   // ANCHOR: lnurl-withdraw
-  /// Endpoint can also be of the form:
-  /// lnurlp://domain.com/lnurl-pay?key=val
-  /// lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4excttsv9un7um9wdekjmmw84jxywf5x43rvv35xgmr2enrxanr2cfcvsmnwe3jxcukvde48qukgdec89snwde3vfjxvepjxpjnjvtpxd3kvdnxx5crxwpjvyunsephsz36jf
-  String lnurlPayUrl = "lightning@address.com";
+  // Endpoint can also be of the form:
+  // lnurlw://domain.com/lnurl-withdraw?key=val
+  String lnurlWithdrawUrl = "lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4exctthd96xserjv9mn7um9wdekjmmw843xxwpexdnxzen9vgunsvfexq6rvdecx93rgdmyxcuxverrvcursenpxvukzv3c8qunsdecx33nzwpnvg6ryc3hv93nzvecxgcxgwp3h33lxk";
 
-  InputType inputType = await BreezSDK().parseInput(input: lnurlPayUrl);
-  if (inputType is InputType_LnUrlWithdraw) {
-    int amountMsat = inputType.data.minWithdrawable;
-    LnUrlWithdrawRequest req = LnUrlWithdrawRequest(
-      data: inputType.data,
-      amountMsat: amountMsat,
-      description: "<description>",
-    );
-    LnUrlWithdrawResult result = await BreezSDK().lnurlWithdraw(req: req);
-    print(result.data);
+  try {
+      InputType inputType = await parseInput(s: lnurlWithdrawUrl);
+      if (inputType is InputType_LnUrlWithdraw) {
+          int amountSats = inputType.data.minWithdrawable;
+          LnUrlCallbackStatus result = await lnurlWithdraw(
+              reqData: inputType.data,
+              amountSats: amountSats,
+              description: "comment",
+          );
+      }
+  } catch (error) {
+      // handle error
```

Comment:
There's no need to update this latter part.

Reply:
True!

I was confused by the `lnurlPayUrl` in the first line, will revert.

Reply:
Done.